src/Compilers/Core/Desktop/AnalyzerFileReference.InMemoryAssemblyLoader.cs

```diff
@@ -150,14 +150,20 @@ private static Assembly LoadCore(string fullPath)
             /// </summary>
             private static Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEventArgs args)
             {
-                if (args.RequestingAssembly == null)
-                {
-                    return ResolveForUnknownRequestor(args.Name);
-                }
-                else
+                try
                 {
-                    return ResolveForKnownRequestor(args.Name, args.RequestingAssembly);
+                    if (args.RequestingAssembly == null)
+                    {
+                        return ResolveForUnknownRequestor(args.Name);
+                    }
+                    else
+                    {
+                        return ResolveForKnownRequestor(args.Name, args.RequestingAssembly);
+                    }
                 }
+                catch { }
+
+                return null;
```

Comment:
Will something else report an exception in such cases if the resolution fails in subsequent handlers / CLR? Or does this mean that it is possible for the error to never be reported?